src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import * as api from "../api";
+
+export default function ArticleVotes({ articleCard }) {
+  const [upvoted, setUpvoted] = useState(false);
+  const [resetUpvote, setResetUpvote] = useState(null);
+  const [downvoted, setDownvoted] = useState(false);
+  const [resetDownvote, setResetDownvote] = useState(false);
```

Comment:
You have 4 pieces of state to keep track of whether they've voted up or down. This works but could be simpler as your `upvoted` and `downvoted` state is enough to work out whether you can reset up or down. Consider simplifying this.